### PR TITLE
Rm housing evals

### PR DESCRIPTION
**Pull request by mxmeinhold**
Check one:
- [x] Semantic Change: something about the meaning of the text is different
- [ ] Non-semantic Change: Spelling, grammar, or formatting changes.

Summary of change(s):

Deletes bad references to Housing Evals, makes housing points automatic. Also updates constitution to current process.

### CODE REVIEW

**Comment by saucetray** [comment]
LGTM 🎆 🍾

**Comment by nicholastmosher** [comment]
In theory this should fix the state of the points system, but in practice this will only work starting with incoming freshmen. I think that housing points should not need to be "accumulated", but instead that they should simply be a property of your membership that is derived from the number of years you've been an active member. This would make a difference in a case like myself, where I have already "missed" housing points in the past.

* Under this proposed amendment as-is, I would - from this point forward - always receive my 2 housing points each year. However, since I "missed" housing points in the past under the old system, I still only have 4 housing points.
* Under the "derived points" system, the number of points I have is simply calculated from the number of years I've been a member (i.e. years since passing my first spring evals). If the calculation is simply "number of years as a member * 2", then I would have (8 points? 10 points? depending on whether points get added at the beginning or end of the year)

I'm in favor of a "derived points" system because it eliminates the edge case where people missed housing points from the old (and broken) points system.

File: constitution.tex [hunk 1]
```diff
@@ -673,11 +673,11 @@
...
 The member with top priority is the member with on-floor status and the most Housing Priority Points.
-Housing points are accumulated during the Housing Evaluation described in \ref{Housing Evaluation}.
+Housing Priority Points are accumulated once per Operating Session at the conclusion of the Membership Evaluation. Each Active Member with On-Floor Status who passes the Membership Evaluation is granted two (2) Housing Priority Points.
 \\* \\*
 In the event of a tie, the members will be approached simultaneously and if they are unable to decide fairly between themselves, the assignment of priority will be made by random selection of the tied members.
 \asubsubsubsection{Single Room Assignments}
-When a single room on the House becomes available it is offered to the member who carries the highest Housing Priority as defined in By-Law V, Section H.
+When a single room on the House becomes available it is offered to the member who carries the highest Housing Priority as defined in \ref{Housing Priority System}.
```
Comment (saucetray) [inline constitution.tex]: What if two people have the same housing priority

Comment (mxmeinhold) [reply constitution.tex]: Copy the line about ties from just above?

**Comment by mxmeinhold** [comment]
Handed out: 61
Yes: 58
No: 2
Abstain: 1

**Pull request merged**